source/api/cypress-api/isbrowser.md

```diff
@@ -2,35 +2,87 @@
 title: Cypress.isBrowser
 ---
 
-`Cypress.isBrowser` returns `true` if the current browser matches the filter passed or `false` if it does not.
+`Cypress.isBrowser` checks if the current browser matches the given name or filter.
 
 # Syntax
 
 ```javascript
-Cypress.isBrowser('chrome') // true when running in Chrome
-Cypress.isBrowser('firefox') // true when running in Firefox
-Cypress.isBrowser({ family: 'chromium' }) // true when running in any Chromium-based browser (Chrome, Electron, etc...)
-Cypress.isBrowser({ family: 'chromium', channel: 'stable' }) // true when running in any stable release of a Chromium-based browser
+Cypress.isBrowser(name)
+Cypress.isBrowser(filter)
 ```
 
 ## Arguments
 
-**{% fa fa-angle-right %} filter**  ***(string or object)***
+**{% fa fa-angle-right %} name**  ***(String)***
 
-The filter to check against. If an object, it should match the browser types returned by the {% url "browser launch API" browser-launch-api %}. If a name, it will be matched against the name of the browser (case-insensitive).
+The name of the browser (case-insensitive).
+
+**{% fa fa-angle-right %} filter**  ***(Object)***
+
+Filter by the browser properties. You can inspect the current browser properties by using the {% url "Browser Launch API" browser-launch-api %}. Supported properties are:
```

Comment:
can we link to https://docs.cypress.io/api/cypress-api/browser.html instead of the `before:browser:launch` docs here, since this is in the `driver`?

```suggestion
Filter by the browser properties. You can inspect the current browser's properties by looking at {% url "`Cypress.browser`" browser %}.
```

and then remove the list of properties below